preload/dynamic_remove_preload_href.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+    var t = async_test('Makes sure that dynamically removed preloaded resource stop downloading');
+</script>
+<body>
+<script>
+    var link = document.createElement("link");
+    link.as = "script";
+    link.rel = "preload";
+    link.href = "resources/dummy.js?pipe=trickle(d0.3)";
+    document.body.appendChild(link);
+    link.href = "";
+    window.addEventListener("load", t.step_func(function() {
+        assert_equals(performance.getEntriesByType("resource").length, 3);
+        t.done();
```

Comment:
Not sure I follow this test. How does this verify that the preload response is aborted? Meta: does abort even make sense? The browser should start a new download if href is mutated, but it's not clear that the old one can/should be stopped.